javascript/javascript1/week2/main.js

```diff
@@ -0,0 +1,143 @@
+//Flight booking fullname function
+function getFullname(firstname, surname, gender, useFormalName) {
+	if (gender === "male" && useFormalName === true) {
+		return "Your client full name is Lord " + firstname + " " + surname;
+	} else if (gender === "female" && useFormalName === true) {
+		return "Your client full name is Queen " + firstname + " " + surname;
+	} else if (gender === "" && useFormalName === true) {
+		return "Your client full name is " + firstname + " " + surname;
+	} else {
+		return "Please provide valid information";
+	}
+}
+console.log(getFullname("Joy", "Kumar", "male", true));
+console.log(getFullname("Riya", "Kumari", "female", true));
+console.log(getFullname("Oscar", "Roy", "", true));
+console.log(getFullname("", "", "", false));
+
+//End of Flight booking fullname function
+
+//Event application
+function getEventWeekday(days) {
+	const weekdays = [
```

Comment:
This is fine if we are calling the function once on the site. however, the function helps to reuse the same code again and again with a different parameter. In such a case you will be defining generic variables again and again on the page. for ex 'weekdays' will assign again and again whenever you call this function on site.  though logically you are not playing with this in your function

Reply:
do you mean i have to put that weedays array out of function.

Reply:
yes